core/explorer.go

```diff
@@ -40,18 +50,32 @@ type ExplorerMapDID struct {
 	PeerID string `json:"peer_id"`
 }
 
-type ExplorerTrans struct {
-	TID         string   `json:"transaction_id"`
-	SenderDID   string   `json:"sender_did"`
-	ReceiverDID string   `json:"receiver_did"`
-	TokenTime   float64  `json:"token_time"`
-	TokenIDs    []string `json:"token_id"`
-	Amount      float64  `json:"amount"`
-	TrasnType   int      `json:"transaction_type"`
-	QuorumList  []string `json:"quorum_list"`
-	DeployerDID string   `json:"deployer_did"`
-	ExecutorDID string   `json:"executor_did"`
-	//BlockHash   string   `json:"block_hash"`
+type Token struct {
+	TokenID    string  `json:"token_id"`
+	TokenValue float64 `json:"token_value"`
+}
+
+type ChildToken struct {
+	TokenID    string  `json:"child_token_hash"`
```

Comment:
can you update the TokenID as ChildTokenID

Reply:
Done, is TokenValue fine? Also, the json string is different in the latest swagger. So udpated that as well.